clients/src/main/java/org/apache/kafka/common/requests/CreateAclsRequest.java

```diff
@@ -54,16 +54,16 @@
                     PERMISSION_TYPE))));
 
     /**
-     * Version 1 adds RESOURCE_NAME_TYPE.
-     * Also, when the quota is violated, brokers will respond to a version 1 or later request before throttling.
+     * Version 1 adds RESOURCE_PATTERN_TYPE, to support more than just literal resource patterns.
+     * For more info, see {@link PatternType}.
```

Comment:
Does the JavaDoc work without the full class path?

Reply:
Unlike ScalaDoc, yes, as long as the type is imported, which it is.